src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -131,17 +125,17 @@ describe('<PillButton> component', () => {
 
         it('should render correctly', () => {
             expect(view.container).toMatchSnapshot();
-            expect(screen.getByRole('button', { name: '' })).not.toHaveClass(
+            expect(getButtonByName('')).not.toHaveClass(
```

Comment:
s/a there are more like this in the file

Reply:
![image](https://github.com/textkernel/oneui/assets/104063210/58337a77-adbb-4d2f-9cba-7fb63297e53c)

Reply:
s/a